code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -37,7 +37,7 @@
 		if(M.reagents)
 			var/modifier = Clamp((1 - touch_protection), 0, 1)
 			var/amount = round(reac_volume*modifier, 0.1)
-			if(amount >= 1)
+			if(amount >= 0.5)
```

Comment:
this was meant for another PR, but might as well keep it. It makes reagent absorption though VAPOR method more sensible.